code/__DEFINES/overlays.dm

```diff
@@ -22,7 +22,10 @@
 				AA.copy_overlays(changed_on, TRUE);\
 			}\
 		} \
-	}\
-	if(isturf(changed_on)){SSdemo.mark_turf(changed_on);}\
-	if(isobj(changed_on) || ismob(changed_on)){SSdemo.mark_dirty(changed_on);}\
-	//Monkestation Edit: REPLAYS //I Feel Dirty
+	} \
+	if(isturf(changed_on)) { \
+		SSdemo.mark_turf(changed_on); \
+	} else if(isobj(changed_on) || ismob(changed_on)) { \
+		SSdemo.mark_dirty(changed_on); \
+	}
```

Comment:
if it's a turf, then it's not an obj or mob, so changing to `else if` here should a tiny few cpu cycles

also formatted it better so it's less of an eyesore